sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2022-2023.bpmn

```diff
@@ -841,17 +841,18 @@ dmnFullTimeIncomeThreshold</bpmn:text>
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=if studentDataLivingathomeRent = null&#10;then &#34;no&#34;&#10;else&#10;  studentDataLivingathomeRent" target="studentDataLivingathomeRent" />
-          <zeebe:output source="=if studentDataTransportationCost = null&#10;then 0&#10;else&#10;  studentDataTransportationCost" target="studentDataTransportationCost" />
```

Comment:
Why is `studentDataTransportationCost` removed ?

Reply:
As we talked studentDataTransportationCost was replaced to studentDataReturnTripHomeCost. I added a [comment ](https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/4362#issuecomment-2711406960)to the ticket.

Reply:
Thanks for confirming. In that case, can we also remove the `studentDataTransportationCost` from load-assessment-consolidated-data ?


![image](https://github.com/user-attachments/assets/c84994a8-b138-40e3-9b65-efe9079900e1)

Reply:
I confirmed with Biz. A new AC has been added: 
"Remove from consolidated data 'studentDataTransportationCost' (it is no longer related to any form questions)"
There's a `transportationCost` saved to `workflow_data` that's the total transportation cost.